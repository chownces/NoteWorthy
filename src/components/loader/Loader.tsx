import { Loader as SemanticLoader } from 'semantic-ui-react';

const Loader: React.FC = () => {
  return <SemanticLoader active>Loading...</SemanticLoader>;
};

export default Loader;
