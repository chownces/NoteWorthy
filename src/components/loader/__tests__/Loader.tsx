import { render } from '@testing-library/react';

import Loader from '../Loader';

test('Loader component loads correctly', () => {
  const { container: tree } = render(<Loader />);
  expect(tree).toMatchSnapshot();
});
