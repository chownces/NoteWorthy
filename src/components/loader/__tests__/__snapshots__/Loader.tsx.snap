// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component loads correctly 1`] = `
<div>
  <div
    class="ui active text loader"
  >
    Loading...
  </div>
</div>
`;
